master/reference/public-cloud/digital-ocean.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Calico on Digital Ocean
+---
+
+Calico is designed to provide high performance massively scalable virtual
+networking for containers in public cloud environments such as Digital Ocean.
+
+## How to Run Calico in Digital Ocean
+
+Digital Ocean offers two networking options for droplets.
+Calico can be launched on either:
+
+- Public Interface
+- Private Interface [Optional]
+
+We recommend running Calico on the private interface, as it enhances security by
+ensuring Calico services are not accessible from the wider internet.
+
+However, hosts in different Datacenter Regions will not have IP connectivity with one
+another on their private address, and therefore will not be able to establish
+BGP sessions with one another. For mluti-region digital ocean clusters,
+bind Calico to the public interface.
+
+###### Enable Encapsulation
+
+In Digital Ocean, cross-host container-to-container traffic will travel over an
+L3 hop. Since Digital Ocean does not allow peering to the L3 hop, it will
```

Comment:
"L3 hop" doesn't sound right. There could be more than 1 L3 hops involved in this.  Maybe call it "L3 network"?